yt/data_objects/tests/test_time_series.py

```diff
@@ -34,71 +35,115 @@ def test_no_match_pattern():
         )
 
 
-def test_init_fake_dataseries():
-    file_list = [f"fake_data_file_{str(i).zfill(4)}" for i in range(10)]
+@pytest.fixture
+def FakeDataset():
+    class __FakeDataset(Dataset):
```

Comment:
In Python, it's preferable to avoid naming anything with leading double underscores (and without trailing double underscores), because of [how name mangling works](https://docs.python.org/3/tutorial/classes.html#private-variables)
```suggestion
    class _FakeDataset(Dataset):
```